applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_small_strain_u_pw_element.cpp

```diff
@@ -0,0 +1,545 @@
+        "DISPLACEMENT_Y", "WATER_PRESSURE", "WATER_PRESSURE", "WATER_PRESSURE"};
+
+    KRATOS_EXPECT_EQ(variable_names, desired_variable_list);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(UPwSmallStrainElement_IntegrationMethod, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    const auto p_geometry =
+        std::make_shared<Triangle2D3<Node>>(ModelSetupUtilities::Create2D3NTriangleGeometry());
+    const auto                        p_properties = std::make_shared<Properties>();
+    const UPwSmallStrainElement<2, 3> element(0, p_geometry, p_properties,
+                                              std::make_unique<PlaneStrainStressState>());
+
+    // Act
+    const auto p_integration_method = element.GetIntegrationMethod();
+
+    // Assert
+    constexpr auto expected_integration_method = GeometryData::IntegrationMethod::GI_GAUSS_2;
+    KRATOS_EXPECT_EQ(p_integration_method, expected_integration_method);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(UPwSmallStrainElement_CheckDoesNotThrowOnCorrectInput, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    Model model;
+    auto  element = CreateUPwSmallStrainElementWithUPwDofs(model, CreateProperties());
+    SetSolutionStepValuesForGeneralCheck(element);
+    auto process_info = ProcessInfo{};
+
+    // Act, no exceptions on correct input
+    KRATOS_EXPECT_EQ(element->Check(process_info), 0);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(UPwSmallStrainElement_CalculatesSteadyStateRightHandSide, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    Model model;
+    auto  element = CreateUPwSmallStrainElementWithUPwDofs(model, CreateProperties());
+    SetSolutionStepValuesForFluidFluxCheck(element);
+    auto process_info = ProcessInfo{};
+
+    // Act
+    element->Initialize(process_info);
+    Vector actual_right_hand_side;
+    element->CalculateRightHandSide(actual_right_hand_side, process_info);
+
+    // Assert
+    Vector expected_right_hand_side = ZeroVector(9);
+    expected_right_hand_side[7]     = -4.542;
+    expected_right_hand_side[8]     = +4.542;
+    KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(actual_right_hand_side, expected_right_hand_side, Defaults::relative_tolerance);
+    std::vector<array_1d<double, 3>> calculated_fluid_flux_at_integration_points;
+    element->CalculateOnIntegrationPoints(
+        FLUID_FLUX_VECTOR, calculated_fluid_flux_at_integration_points, process_info);
+    for (const auto& calculated_fluid_flux : calculated_fluid_flux_at_integration_points) {
+        KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(
+            calculated_fluid_flux, (array_1d<double, 3>{0., 9.084, 0.}), Defaults::relative_tolerance);
+    }
+}
+
+KRATOS_TEST_CASE_IN_SUITE(UPwSmallStrainElement_CalculatesSteadyStateLeftHandSide, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    Model model;
+    auto  element = CreateUPwSmallStrainElementWithUPwDofs(model, CreateProperties());
+    SetSolutionStepValuesForGeneralCheck(element);
+    auto process_info = ProcessInfo{};
```

Comment:
```suggestion
    const auto process_info = ProcessInfo{};
```

This suggestion applies to some other unit tests, too.

Reply:
done